modules/deploy-red-hat-openshift-container-storage.adoc

```diff
@@ -9,57 +9,57 @@
 |See the following {rh-storage-first} documentation:
 
 |What's new, known issues, notable bug fixes, and Technology Previews
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/4.12_release_notes[OpenShift Data Foundation 4.12 Release Notes]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/4.13_release_notes[OpenShift Data Foundation 4.13 Release Notes]
```

Comment:
This link doesn't work - ODF 4.13 doesn't exist yet. Not sure if it'll go out with OCP 4.13 GA, but it might be after. We might not be able to update these items now